### PR TITLE
Update privacy_policy.html.erb

### PR DESCRIPTION
**Pull request by EmmaFrith**
copy edit to PP

### CODE REVIEW

File: app/views/pages/privacy_policy.html.erb [hunk 1]
```diff
@@ -12,123 +12,98 @@
...
+      <li>validating your teacher reference number (TRN)</li>
+      <li>working out any funding you’re eligible for</li>
+      <li>getting in touch with you and NPQ providers about your NPQ registration</li> 
+      <li>determining NPQ outcomes</li>
+      <li>comparing your data to other data we hold</li>
       <li>getting in touch if there’s a security issue concerning your data</li>
-      <li>getting in touch with candidates and teacher training providers about applications</li>
-      <li>Determining outcomes of teacher training</li>
-      <li>Match, link and analyse your data along with other datasets</li>
-    </ul>
```
Comment (peteryates) [inline app/views/pages/privacy_policy.html.erb:105]: List needs closing here with an `</ul>` or the subsequent content is indented

image

File: app/views/pages/privacy_policy.html.erb [hunk 2]
```diff
@@ -178,7 +151,7 @@
       We also use Google’s G Suite to process some personal data. Google processes your data according to our instructions.
     </p>
 
-    <h4 class="govuk-heading-s">Hosting services</h4>
+    <h3 class="govuk-heading-s">Hosting services</h3>
```
Comment (peteryates) [inline app/views/pages/privacy_policy.html.erb:154]: I think this should be a `govuk-heading-m` as it's smaller than the equally-levelled heading above ☝🏽

File: app/views/pages/privacy_policy.html.erb [hunk 3]
```diff
@@ -187,31 +160,25 @@
...
-
-    <h4 class="govuk-heading-s">Evaluators</h4>
-
-    <p class="govuk-body">
-      We use external evaluators to understand the effectiveness of the teacher training you are receiving from teacher training providers and the impact it is having on the teacher workforce.
+      We use external auditors to quality assure NPQ providers to ensure that you’re receiving the appropriate standard of training.
     </p>
 
-    <h4 class="govuk-heading-s">Appropriate bodies</h4>
+    <h3 class="govuk-heading-s">Evaluators</h3>
```
Comment (peteryates) [inline app/views/pages/privacy_policy.html.erb:166]: This too.

File: app/views/pages/privacy_policy.html.erb (hunk 2 above)
Comment (EmmaFrith) [inline app/views/pages/privacy_policy.html.erb:154]:
```suggestion
    <h3 class="govuk-heading-m">Hosting services</h3>
```

File: app/views/pages/privacy_policy.html.erb [hunk 4]
```diff
@@ -187,31 +160,25 @@
     <h4 class="govuk-heading-s">Auditors</h4>
```
Comment (EmmaFrith) [inline app/views/pages/privacy_policy.html.erb:160]:
```suggestion
    <h3 class="govuk-heading-m">Auditors</h3>
```

File: app/views/pages/privacy_policy.html.erb [hunk 5]
```diff
@@ -12,123 +12,98 @@
...
-      <li>validating teacher reference numbers (TRN)</li>
-      <li>working out any funding which teacher training providers are entitled to</li>
+    
+      <li>sending your registration details to NPQ providers</li>
+      <li>validating your teacher reference number (TRN)</li>
+      <li>working out any funding you’re eligible for</li>
+      <li>getting in touch with you and NPQ providers about your NPQ registration</li> 
+      <li>determining NPQ outcomes</li>
+      <li>comparing your data to other data we hold</li>
       <li>getting in touch if there’s a security issue concerning your data</li>
```
Comment (EmmaFrith) [inline app/views/pages/privacy_policy.html.erb:75]:
```suggestion
      <li>getting in touch if there’s a security issue concerning your data</li>
      </ul>
```

File: app/views/pages/privacy_policy.html.erb [hunk 6]
```diff
@@ -178,40 +152,34 @@
...
-    <h4 class="govuk-heading-s">Evaluators</h4>
+    <h3 class="govuk-heading-m">Auditors</h3>
 
     <p class="govuk-body">
-      We use external evaluators to understand the effectiveness of the teacher training you are receiving from teacher training providers and the impact it is having on the teacher workforce.
+      We use external auditors to quality assure NPQ providers to ensure that you’re receiving the appropriate standard of training.
     </p>
 
-    <h4 class="govuk-heading-s">Appropriate bodies</h4>
+    <h3 class="govuk-heading-s">Evaluators</h3>
```
Comment (EmmaFrith) [inline app/views/pages/privacy_policy.html.erb:167]:
```suggestion
    <h3 class="govuk-heading-m">Evaluators</h3>
```

File: app/views/pages/privacy_policy.html.erb [hunk 7]
```diff
@@ -12,123 +12,99 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">Privacy Policy</h1>
```
Comment (EmmaFrith) [inline app/views/pages/privacy_policy.html.erb:14]:
```suggestion
    <h1 class="govuk-heading-xl">Privacy policy</h1>
```

File: app/views/pages/privacy_policy.html.erb [hunk 8]
```diff
@@ -11,124 +11,100 @@
...
-      <li>your teacher reference number (TRN)</li>
-      <li>your date of birth</li>
-      <li>your National Insurance number</li>
-      <li>where you work</li>
-      <li>what your role is</li>
+    <ul class="govuk-list govuk-list--bullet">   
+     <li>name</li>
+     <li>phone number</li>
+     <li>email address</li>
+     <li>teacher reference number (TRN</li>
```
Comment (EmmaFrith) [inline app/views/pages/privacy_policy.html.erb:51]:
```suggestion
     <li>teacher reference number (TRN)</li>
```

File: app/views/pages/privacy_policy.html.erb [hunk 9]
```diff
@@ -228,32 +196,32 @@
...
-      You can also use our contact form to <%= govuk_link_to "get in touch with our Data Protection Officer", "https://form.education.gov.uk/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-f1453496-7d8a-463f-9f33-1da2ac47ed76/AF-Stage-1e64d4cc-25fb-499a-a8d7-74e98203ac00/definition.json&redirectlink=%2Fen&cancelRedirectLink=%2Fen" %>.
-    </p>
-
-    <p class="govuk-body">
-      Once an applicant has enrolled with a teacher training provider, we will not be able to ask them to remove personal data from their systems. Please contact the provider separately.
+      <%= govuk_link_to "Contact the Department for Education", "https://form.education.gov.uk/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-f1453496-7d8a-463f-9f33-1da2ac47ed76/AF-Stage-1e64d4cc-25fb-499a-a8d7-74e98203ac00/definition.json&redirectlink=%2Fen&cancelRedirectLink=%2Fen" %> if you want to ask us to remove your data or get access to the data we have about you.
     </p>
+      
+    <p class="govuk-body"> 
+      Please contact the NPQ providers regarding removing your data from their systems.
```
Comment (EmmaFrith) [inline app/views/pages/privacy_policy.html.erb:206]:
```suggestion
      Contact NPQ providers separately regarding removing your data from their systems. 
```

File: app/views/pages/privacy_policy.html.erb (hunk 2 above)
Comment (EmmaFrith) [reply app/views/pages/privacy_policy.html.erb]: thank you <3

**Review by peteryates — APPROVED** [review]
Looks good, definitely more readable.

Perhaps we should link to the documents we reference (GDPR, Data Protection Act).

So long as the team's happy the wording is changed and not the _meaning_ I'm happy for this to go in.

**Pull request merged**